README.md

```diff
@@ -67,8 +69,11 @@ app.use('/api/v1/', routes)
 
 ## Tasks
 * `git clone` this repository, run `npm install`.
-*  Run `createdb fac-express` - **IMPORTANT**
-*  Setup the database by running `npm run build:db` - **IMPORTANT**
+* Set up the database:
+  * Run `createdb fac-express`
+  * Run `psql` or `pgcli` to enter the pg interactive terminal.
+  * Enter `CREATE USER super WITH SUPERUSER PASSWORD 'password';`
+  * Run `npm run build:db`
```

Comment:
I think we probably need to add something about getting out of psql before running the npm script: 
>Enter `\q` in psql to leave the psql interactive terminal

Reply:
All done 👍